---
geometry_msgs/Point current_position
float32[] current_joints
